src/components/Header/styles/index.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+import Content from './content';
+import Thumbnail from './thumbnail';
+import Title from './title';
+import Subtitle from './subtitle';
+import Icon from './icon';
+
+const StyledHeader = styled.div`
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  padding-top: 2px;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
```

Comment:
Could you remove those browser prefixes please? The styled-components library is already in charge of adding them automatically for us.